text/0035-opentelemetry-protocol.md

```diff
@@ -82,23 +82,76 @@ If the client is shutting down (e.g. when the containing process wants to exit)
 
 If the client is unable to deliver a certain request (e.g. a timer expired while waiting for acknowledgements) the client SHOULD record the fact that the data was not delivered.
 
-#### Result Code
+#### Export Response
 
-`Export` response includes a `result_code` field, which indicates whether the server was able to successfully process the received data. Possible values for `result_code` field are:
+The server may respond with either a success or an error to export requests.
 
-- `Success` - telemetry data is successfully processed by the server. If the server receives an empty request (a request that does not carry any telemetry data) the server SHOULD respond with `Success`.
+The success response indicates telemetry data is successfully processed by the server. If the server receives an empty request (a request that does not carry any telemetry data) the server SHOULD respond with success.
```

Comment:
Should this say that the data _was processed_ by the server or that the server successfully accepted it for processing (but did not necessarily finish yet)?

Reply:
I think that's beyond the scope of the protocol. 

Responding with success means "I am now responsible for the data, don't worry about it anymore". What exactly that means for the particular server is that particular server's business. Some servers may reply immediately after deserializing the request, some others may do it after writing to disk and fsync-ing.

The protocol doesn't care about it, the protocol only provides means for the client and server to agree on who is responsible for the data after the request finishes.

This is I believe the typical separation of concerns between protocol and application.

Reply:
Thought so. In this case it would make sense to reword it to "received" or "accepted" (for processing) to make that clear.

```suggestion
The success response indicates telemetry data was successfully received by the server. If the server receives an empty request (a request that does not carry any telemetry data) the server SHOULD respond with success.
```

Reply:
I don't think "received" conveys the intent. The server may "receive" the data, then return non-success response.

"successfully accepted" does not seem to be a better wording either (is there an "unsuccessful acceptance"?).